modules/serverless-rn-1-24-0.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-24-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.24.0
+
+{ServerlessProductName} 1.24.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.24.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.3.
+* {ServerlessProductName} now uses Knative Eventing 1.3.
+* {ServerlessProductName} now uses Kourier 1.3.
+* {ServerlessProductName} now uses Knative `kn` CLI 1.3.
+* {ServerlessProductName} now uses Knative Kafka 1.3.
+* The `kn func` CLI plug-in now uses `func` 0.24.
+
+* Init containers support for Knative services is now generally available (GA).
+* FIPS is now disabled for the `KafkaBroker`, `KafkaSource` and `KafkaSink` components.
+
+ifdef::openshift-enterprise[]
+* You can now use the link:serverless/integrations/serverless-cost-management-integration.html#serverless-cost-management-integration_serverless-cost-management-integration[cost management service] with {ServerlessProductName}.
+endif::[]
+
+[id="fixed-issues-1.24.0_{context}"]
+== Fixed issues
+
+* The security issue, where a malicious actor could do a Denial-of-Service (DOS) attack on a server using the `InstrumentHandlerCounter` function in `prometheus/client_golang`, has been fixed. (CVE-2022-21698)
```

Comment:
```suggestion
* In previous versions of {ServerlessProductName}, it was possible for a malicious actor to cause a Denial-of-Service (DOS) attack on a server, by using the `InstrumentHandlerCounter` function in `prometheus/client_golang`. This issue has been fixed (CVE-2022-21698).
```

suggested format for fixed issues

Reply:
Deleted the CVE description altogether, per discussions.